### ISSUE TITLE
To support panoptic segmentation dataset format

### ISSUE THREAD

**Issue by sailfish009** [heartexlabs/label-studio#6322]
**Is your feature request related to a problem? Please describe.**
I haven't seen any opensource annotation tools that support panoptic segmentation dataset

**Describe the solution you'd like**
I would like to be able to annotate a panoptic segmentation dataset in Label Studio.

**Describe alternatives you've considered**
I'm not sure, but I think I read in the CVAT issue thread that some teams are able to do it manually.

**Additional context**
[CVAT issue](https://github.com/cvat-ai/cvat/issues/8404)

**Comment by userwatch** [comment]
Hello @sailfish009 , I am trying to train Panoptic with my custom data. How did you create the panoptic annotation?  Did you solved it?
Thanks:)

**Comment by sailfish009** [comment]
@userwatch, No, you won't be able to use it for panoptic segmentation unless it's a commercial program.
In practice,  name and  bbox is enough, plus I find label-studio to be more flexible and less buggy than cvat.
